dashboard.css

```diff
@@ -0,0 +1,328 @@
+body {
+    font-family: -apple-system, BlinkMacSystemFont, Segoe UI, Helvetica, Arial, sans-serif, Apple Color Emoji, Segoe UI Emoji;
+    width: 100%; /* edit : Changed from 1000px to adjust screen display */
+    background-color: #eeeeee;
+}
+
+a {
+    text-decoration: none;
+    color: #0366d6;
+}
+
+:root {
+    --color-red-background: #ee0000;
+    --color-red-foreground: #ffffff;
+    --color-red-border: #cc0000;
+
+    --color-yellow-background: #eeee00;
+    --color-yellow-foreground: #000000;
+    --color-yellow-border: #cccc00;
+
+    --color-green-background: #00bb00;
+    --color-green-foreground: #ffffff;
+    --color-green-border: #999900;
+
+    --color-blue-background: #0000ee;
+    --color-blue-foreground: #ffffff;
+    --color-blue-border: #0000cc;
+
+    --color-black-background: #000000;
+    --color-black-foreground: #ffffff;
+    --color-black-border: #333333;
+}
+
+/* edit : Added to have a background color */
```

Comment:
Ideally we could add our custom overriding styles  in another file, so that the base one can be easily replaced if there is a newer one in dashboard repo. Not blocking for this PR, but can we track and do this after we have the first iteration?

Reply:
The html file that gets generated refers to these classes.
That is an automated process. So having separate css won't help because everytime the HTML file gets generated, it will have these classes referred.